Source/WebCore/rendering/EventRegion.cpp

```diff
@@ -209,8 +209,42 @@ bool EventRegionContext::shouldConsolidateInteractionRegion(IntRect bounds, Rend
     return false;
 }
 
+void EventRegionContext::addOcclusionMarginsForSmallInteractionRegions()
+{
+    constexpr int minimumSize = 20;
+    constexpr int occlusionMargin = 10;
+    Vector<InteractionRegion> interactionRegionsWithOcclusions;
+    for (auto& interactionRegion : m_interactionRegions) {
+        bool isSmallRect = false;
+        auto occlusionRect = interactionRegion.rectInLayerCoordinates;
+
+        if (interactionRegion.rectInLayerCoordinates.width() < minimumSize) {
+            occlusionRect.inflateX(occlusionMargin);
+            isSmallRect = true;
+        }
+        
+        if (interactionRegion.rectInLayerCoordinates.height() < minimumSize) {
+            occlusionRect.inflateY(occlusionMargin);
+            isSmallRect = true;
+        }
+            
+        if (isSmallRect) {
+            interactionRegionsWithOcclusions.append({
+                interactionRegion.elementIdentifier,
+                occlusionRect,
+                interactionRegion.borderRadius,
```

Comment:
not sure, but it might be counter productive to relay the border radius as it could reduce the positive impact of the "shield"